README.md

```diff
@@ -1 +1,88 @@

#### Reviewing code

When reviewing code, you are checking for some things and you are not checking for other things. It’s easy to say what you aren't checking for. 

##### What _not_ to check for

You aren't checking for the most nit picky opinionated stuff that makes you seem more superior and makes their code seem like garbage. This isn't a competition in anyway so by being nit picky and annoying you aren't winning. Be respectful and remember we are all on the same team.

##### What _to_ check for

We are reviewing to make the code better than it already was or validate that it’s the best it’s gonna be. Every comment and review you make should be in the interest of better our code base and teaching other people new things. If you think that your comment is too nit-pick but it might help the developer learn something put it in the form of a suggestion not a dictation. For example, I've gotten this review many times

My code `assert len(some_list) == 0`

The review "Idiomatic Python says we should write that as `assert not some_list`"

When I first got this review, I was annoyed but then realized I just learned something neat and his review was written politely and informatively.

Also, when reviewing point out flaws in convention or style if it has been agreed upon within the group, we are our own police. 


#### Reviewing the acceptance criteria and checking for bugs

When reviewing the app itself we are checking to see that the issue has been solved in its entirety. The question we are asking is did it met the criteria of acceptance and did it introduce any bug. If the answer is "yes" and "no" then we are good. Otherwise this needs to be commented on in the PR, discussed with the developer, and potentially discussed with the team. Before we do this, we need to make sure we are seeing the app as if it were in production (merged with master). 

##### Preparing the code for app review

This is simple as you can just copy these commands and change the issue number accordingly. First, we need to update our repo with remote `git fetch` then we need to checkout a new branch that will be our review branch by basing it off of master `git checkout -b review origin/master`. Finally, we merge this with the feature branch `git merge origin/issue-<#>`. In summary for reviewing an issue numbered 7:

	git fetch
	git checkout -b review origin/master
	git merge origin/issue-7
	
Then you can review the app as if it were already merged with master.


\ No newline at end of file
+# CS 2200 Cache Simulator
+Copyright (c) 2017, Ctrl+Alt+Delicious (Ricky Barillas, Ryan Berst, Eduardo Mestanza, Kevin Rose, Will Thompson)
+
+#### Run the two commands to start the application:
+    <p>     npm install       (only the first time)
+    <p>     npm start
+
```

Comment:
We should remove the `<p>` since they are showing up in the code block

Reply:
weird I didn't notice that before. will do.